<launch>
	<node name="PDcontroller" pkg="rbe_proj" type="joint_controller.py" output="screen"/>
</launch>
